networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/DiscoveryNetwork.java

```diff
@@ -119,6 +124,12 @@ public void addStaticPeer(final String peerAddress) {
     return discoveryService.getEnr();
   }
 
+  public void setLongTermAttestationSubnetSubscriptions(List<Integer> subnetIds) {
```

Comment:
What about:
```suggestion
  public void advertiseLongTermAttestationSubnetSubscriptions(final Set<Integer> subnetIds) {
```

Since the discovery network's role is limited to advertisements.

Reply:
Following on from Meredith's suggestion to use a `Set` in the `Bitvector` constructor, we should probably use a `Collection` here to allow callers to choose either a `List` or `Set`.  The order doesn't matter to us so being flexible is quite useful.

And when we add the piece to call this building up the indices as a `Set` would make a lot of sense.

Reply:
Makes sense.